eslint.config.mjs

```diff
@@ -1,280 +1,221 @@
-            },
-            {
-              from: 'packages/nitro',
-              target: 'packages/!(nitro)/**/*',
-              message: 'nitro should not directly import other packages.'
-            }
-          ]
-        }
-      ],
-      '@typescript-eslint/consistent-type-imports': [
-        'error',
-        {
-          disallowTypeAnnotations: false
-        }
-      ],
+  .override('nuxt/typescript/rules', {
+    rules: {
       '@typescript-eslint/ban-ts-comment': [
         'error',
         {
           'ts-expect-error': 'allow-with-description',
           'ts-ignore': true
         }
       ],
-      '@typescript-eslint/prefer-ts-expect-error': 'error',
+      '@typescript-eslint/no-dynamic-delete': 'off',
       '@typescript-eslint/no-unused-vars': [
         'error',
         {
           argsIgnorePattern: '^_',
-          varsIgnorePattern: '^_',
-          ignoreRestSiblings: true
+          ignoreRestSiblings: true,
+          varsIgnorePattern: '^_'
         }
       ],
-      'jsdoc/check-tag-names': [
-        'error',
-        {
-          definedTags: ['__NO_SIDE_EFFECTS__']
-        }
-      ]
-    },
-    settings: {
-      jsdoc: {
-        ignoreInternal: true,
-        tagNamePreference: {
-          warning: 'warning',
-          note: 'note'
-        }
+      '@typescript-eslint/triple-slash-reference': 'off',
+      '@typescript-eslint/unified-signatures': 'off',
+      ...{
+        // TODO: Discuss if we want to enable this
+        '@typescript-eslint/ban-types': 'off',
+        // TODO: Discuss if we want to enable this
+        '@typescript-eslint/no-explicit-any': 'off',
+        // TODO: Discuss if we want to enable this
+        '@typescript-eslint/no-invalid-void-type': 'off'
       }
     }
-  },
-  {
-    files: ['packages/schema/**'],
+  })
+
+  .override('nuxt/tooling/unicorn', {
     rules: {
-      'jsdoc/valid-types': 'off',
-      'jsdoc/check-tag-names': [
-        'error',
-        {
-          definedTags: ['experimental']
-        }
-      ]
+      'unicorn/no-new-array': 'off',
+      'unicorn/prefer-dom-node-text-content': 'off'
     }
-  },
-  {
-    files: ['packages/nuxt/src/app/**', 'test/**', '**/runtime/**', '**/*.test.ts'],
+  })
+
+  .override('nuxt/vue/rules', {
     rules: {
-      'no-console': 'off'
+
     }
-  },
-  {
-    files: ['test/fixtures/**'],
+  })
+
+  // Stylistic rules
+  .override('nuxt/stylistic', {
     rules: {
-      '@typescript-eslint/no-unused-vars': 'off',
-      'vue/valid-v-for': 'off'
+      // TODO: Discuss if we want to enable this
+      '@stylistic/brace-style': 'off',
+      '@stylistic/indent-binary-ops': 'off',
+      '@stylistic/max-statements-per-line': 'off',
+      '@stylistic/operator-linebreak': 'off',
+      // TODO: Anthony suggests to set this to 'consistent'
+      '@stylistic/quote-props': ['error', 'as-needed'],
+      // TODO: Should we enable this and set to 'never'?
+      '@stylistic/space-before-function-paren': 'off'
```

Comment:
There are a few stylistic rules to discuss; they are not very important, tho.

Reply:
I picked some choices here - we can also discuss later but merging to avoid pending this pr 🙏